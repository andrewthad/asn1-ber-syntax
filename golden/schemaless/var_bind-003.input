3011060D 2B060104 01947801 02070302 000500
