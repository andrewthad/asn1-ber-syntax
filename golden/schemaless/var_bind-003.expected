Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 2
            [ Value
                { tagClass = Universal
                , tagNumber = 6
                , contents = Primitive
                    [ 0x2b
                    , 0x06
                    , 0x01
                    , 0x04
                    , 0x01
                    , 0x94
                    , 0x78
                    , 0x01
                    , 0x02
                    , 0x07
                    , 0x03
                    , 0x02
                    , 0x00
                    ]
                }
            , Value
                { tagClass = Universal
                , tagNumber = 5
                , contents = Primitive []
                }
            ]
        )
    }
